src/Root.tsx

```diff
@@ -1,12 +1,12 @@
-import React from 'react';
 import { List } from './component/list'
-import { Headline } from "./component/headline";
+import { Headline } from './component/headline';
+import * as React from 'react'
```

Comment:
does this change have purpose?)

Reply:
no, I see error during indexing. Currently, I will return it back